apps/backend/src/app/modules/maps/map-submission.service.ts

```diff
@@ -421,6 +422,17 @@ export class MapSubmissionService {
     });
   }
 
+  fileNameChecks(name: string, fileName: string) {
+    // Could probably do some more here, but we want to be relatively lax with
+    // naming, especially as moderators can bring up any issues.
```

Comment:
Should check to see if the file name can be saved out properly? Or do we handle that? I assume S3 probably handles stuff like `^` in the file name just fine but we should check to see if it has file incompatible characters.

Reply:
Oh right this comment is a bit unclear (I'll update it) - we already have the [`@IsMapName` validator](https://github.com/momentum-mod/website/blob/main/libs/backend/validators/src/is-map-name.validator.ts) which only allows alphanumeric, underscores and dashes. We don't E2E test (since E2E testing DTOs is a pain and a million combinations, though I'd really like to), but the validator itself has unit tests.

Reply:
Oh nice I missed that validator.